biocore/micronota
# WIP: big change

The changes include:

1. modify travis to test both linux and osx
2. add uniref database prep
3. add EMBL parser
4. add info command
5. fix the tests of pressers for infernal and hmmer 

I should stop adding more things. @mortonjt , it is ready for review/merge. Travis passed.
